roles/shiftstack/templates/pvc.yml.j2

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ cifmw_shiftstack_client_pod_name }}-pvc
+  namespace: openstack
+spec:
+  storageClassName: local-storage
+  accessModes:
+    - ReadWriteOnce
```

Comment:
Could you please try with ReadWriteMany ? It's arequirement for concurrent shiftstackclient pods